tests/main.js

```diff
@@ -174,7 +168,15 @@ requirejs(['jquery', 'jquery.vellum'], function ($) {
         $('#load-saved').click(function () {
             load(session.getItem("vellum.tests.main.lastSavedForm") || "");
         });
-        load(""); // load empty form on initial page load
+
+
+        // mocha.env is an object when invoked by mocha-phantomjs
+        if (mocha.env) {
```

Comment:
it defers `mocha.run()` until after the first form is fully loaded.

Previously tests were running sort of in parallel with the first form loading, and something, which I wasn't able to identify, is taking a long time (~4s) during that first load. When running all tests, that first group is normally
```
  The plugin manager
    ✓ should ignore unknown plugins (58ms)
```
And as you can see it's not a very long test, but the initialization was hitting the 2000ms timeout.

I said "in parallel" above, but actually "interleaved" is a better word. This interleaving of the first form loading at the same time that tests are running is caused by to use of `setTimeout` in the initialization code.